src/test/java/org/fundacionjala/sfdc/stepdefinition/AccountSteps.java

```diff
@@ -0,0 +1,132 @@
+        SFANewModifyPage newAccountPage = new SFANewModifyPage();
+        CommonActions.setValues(values, newAccountPage.fillMethodsToFields());
+        CommonActions.clickElement(newAccountPage.getSaveNewAccountButton());
+    }
+    /**
+     * Then step.
+     */
+    @Then("^a new account is created$")
+    public void aNewAccountIsCreated(Map<String, String> values) {
+        SFADetailsPage accountDetail = new SFADetailsPage();
+        Assert.assertEquals(accountDetail.getNewAccountSavedName(), values.get("accountName"));
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I choose an account from recent accounts and I click on Edit account$")
+    public void iChooseAnAccountFromRecentAccountsAndIClickOnEditAccount() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        CommonActions.clickElement(accountsMainPage.getAccountNameLink());
+        SFADetailsPage accountsDetailPage = new SFADetailsPage();
+        CommonActions.clickElement(accountsDetailPage.getEditButton());
+    }
+
+    /**
+     * And step.
+     */
+    @And("^I edit the account name field and I press the save button$")
+    public void iEditTheAccountNameFieldAndIPressTheSaveButton() {
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        SFANewModifyPage modifyPage = new SFANewModifyPage();
+        modifyPage.setAccountNameTextField(propertiesManager.getProperty("accountName"));
+        CommonActions.clickElement(modifyPage.getSaveNewAccountButton());
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^an account is modified")
+    public void anAccountIsModified(Map<String, String> values) {
+        SFADetailsPage accountDetail = new SFADetailsPage();
+        Assert.assertEquals(accountDetail.getNewAccountSavedName(), values.get("accountName"));
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I choose last account from recent accounts and I click on Delete account$")
+    public void iChooseLastAccountFromRecentAccountsAndIClickOnDeleteAccount() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        CommonActions.clickElement(accountsMainPage.getAccountNameLink());
+        SFADetailsPage accountsDetailsPage = new SFADetailsPage();
+        CommonActions.clickElement(accountsDetailsPage.getDeleteButton());
+    }
+
+    /**
+     * Then step.
+     */
+    @And("^I click on OK$")
+    public void iClickOnOK() {
+        SFADetailsPage accountsDetailsPage = new SFADetailsPage();
+        accountsDetailsPage.clickDeleteAlert();
+    }
+
+    /**
+     * Then step.
+     */
+    @Then("^the system deletes the account$")
+    public void theSystemDeletesTheAccount() {
+        SFAMainPage accountsMainPage = new SFAMainPage();
+        Assert.assertEquals(accountsMainPage.getAccountHomePage(), "Home");
+    }
+
+    @And("^I fill the fields and press the save button$")
+    public void iFillTheFieldsAndPressTheSaveButton(Map<String, String> values) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "iFillTheAccountNameFieldAndPressTheSaveButton" on line 52. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)